app/components/works/contributor_row_component.html.erb

```diff
@@ -1,5 +1,5 @@
-<div class="contributor row inner-container" data-controller="contributor">
-  <div class="col-md-12">
+<div class="contributors row inner-container" data-controller="contributors" data-action="error->contributors#error" data-target="edit-deposit.contributorsField">
```

Comment:
Do you know if we're using this `contributors` (or `contributor`) class, or if it's a placeholder?

Reply:
Does not appear to be used - I tested without the class and see no difference.  Will just remove to get rid of cruft.